pyomo/contrib/pyros/pyros.py

```diff
@@ -101,50 +101,50 @@ def pyros_config():
     # ================================================
     # === Options common to all solvers
     # ================================================
-    '''CONFIG.declare('time_limit', ConfigValue(
+    CONFIG.declare('time_limit', ConfigValue(
         default=None,
-        domain=NonNegativeFloat
+        domain=NonNegativeFloat, visibility=10
```

Comment:
Why are you specifying `visibility=10`?  That is taking what look like standard user options and marking them as "really, really advanced"?

Reply:
I realized that you can set visibility to accomplish what we wanted with the sphinx documentation. Since we have some config args that are now also positional arguments, setting the visibility of the config to anything > 0 will not display it in the config documentation (removing the redundant documentation.) I picked 10 sort of arbitrarily. Is this okay?

Reply:
I think I accidentally resolved this comment. We don't use visibility to "hide" the positional arguments in the kwargs documentation rendered in pyros.rst, but I think we still need to figure out how to do this programmatically because the positional arguments are now listed as both "parameters" and "keyword arguments".

Reply:
I just pushed a change that adds in a filter before updating the docstring.  This should be resolved now.